sdk/iot/Azure.Iot.Hub.Service/src/Customized/Models/DeviceIdentityData.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Iot.Hub.Service.Models
+{
+    /// <summary>
+    /// Device identity data.
+    /// </summary>
+    [CodeGenModel("Device")]
+    public partial class DeviceIdentityData
```

Comment:
Why the "Data" postfix? Aren't all strong types Data?

Reply:
I was in favor of DeviceData but I see a lot of comments say Identity. So to me it seemed like "data about identifying a device" and hence the name. Correct me if my understanding is wrong.

Reply:
I prefer `DeviceIdentity` instead.

Reply:
I somehow cannot wrap my head around the word Identity. It might just be the different backgrounds we have. However we should be doing the right thing for the customer. Can you explain to me why "DeviceIdentity" is better? What is this data actually defining? It will help us make the right choice.

Reply:
From the service's perspective, they maintain a registry of identities that customers can connect to the service as. For instance, you can register a device identity if you want to connect to the cloud using deviceClient. Likewise, you register a module identity before communicating with the cloud on your ModuleClient instance

Reply:
In creating an identity, you are outlining important connection time information such as authentication type, and if the device is disabled/enabled